### ISSUE TITLE
NullReferenceException: Object reference not set to an instance of an object

### ISSUE THREAD

**Issue by t0chas** [neuoy/KSPTrajectories#66]
Hi, 

I am trying Trajectories with a hole bunch of other mods and the outputlog is filled with this messages

```
NullReferenceException: Object reference not set to an instance of an object
  at Trajectories.NavBallOverlay.SetDisplayEnabled (Boolean enabled) [0x00000] in <filename unknown>:0 

  at Trajectories.NavBallOverlay.Update () [0x00000] in <filename unknown>:0 
 
(Filename:  Line: -1)

NullReferenceException: Object reference not set to an instance of an object
  at FlightGlobals.get_ActiveVessel () [0x00000] in <filename unknown>:0 

  at Trajectories.AutoPilot.Update () [0x00000] in <filename unknown>:0 
 
(Filename:  Line: -1)
```

**Comment by t0chas** [comment]
here is the full log 

[https://dl.dropboxusercontent.com/u/72963294/KSPPlugins/Trajectories/1.6.0/output_log.txt](url)

**Comment by gnivler** [comment]
The NREs start as soon as the main menu loads, here's a log from stock + Trajectories 1.6.0

[KSP.log.txt](https://github.com/neuoy/KSPTrajectories/files/244256/KSP.log.txt)

**Comment by neuoy** [comment]
As stated on the forum, the first exception (Trajectories.NavBallOverlay.SetDisplayEnabled) is fixed, the second one is not fixed yet but doesn't happen during gameplay. Should be fixed in next release.

**Comment by neuoy** [comment]
This issue is fixed in version 1.6.2
